web/src/enums/TransportType.ts

```diff
@@ -1,7 +1,7 @@
 export enum TransportType {
     PublicTransport = 'public_transport',
     Driving = 'driving',
-    Cycling = 'cycling',
-    Walking = 'walking',
+    Cycling = 'cycling+ferry',
+    Walking = 'ferry',
```

Comment:
Klopt dit? Walking lijkt mij geen ferry.

Reply:
Jep, dit is een oplossing aangedragen door iGeolise. `ferry` neemt namelijk wandeltijd tot ferry mee, dan telt het als een merged type

Reply:
Fair enough